common/dynamicconfig/constants.go

```diff
@@ -1254,12 +1254,12 @@ will wait on workflow lock acquisition. Requires service restart to take effect.
 	)
 	EnableHostHistoryCache = NewGlobalBoolSetting(
 		"history.enableHostHistoryCache",
-		false,
+		true,
```

Comment:
Is this enabled in functional tests?

Reply:
let me check that.

Reply:
It is not enabled in functional tests. But it is not disabled explicitly. So after merging this change, it should be enabled.